projects/observability/src/shared/components/span-detail/tags/span-tags-detail.component.ts

```diff
@@ -16,12 +14,15 @@ import { ScopeQueryParam } from '../../../../pages/explorer/explorer.component';
         <ht-list-view [records]="tagRecords" display="${ListViewDisplay.Plain}" data-sensitive-pii>
           <div class="tag-value" *htListViewValueRenderer="let record">
             <div class="value">{{ record.value }}</div>
-            <ht-explore-filter-link
-              class="filter-link"
-              [paramsOrUrl]="this.getExploreNavigationParams | htMemoize: record | async"
+            <ht-filter-button
+              *htLoadAsync="this.metadata$ as metadata"
+              class="filter-button"
+              [attribute]="this.getFilterAttribute | htMemoize: metadata"
+              [metadata]="metadata"
```

Comment:
Could we keep pushing this down? Can the filter button fetch its own metadata and just accept a scope. Probably just have the filter url service accept a scope and let it fetch metadata.

Reply:
Finalize, MetadataService was used in both places

Reply:
No, not possible because they are placed in different routes filterButton in components and MetadataService under observability and move FilterButton means to move more components related. A way to fix this would be to add the method to MetadataService returning FilterAttribute[] or create a new service just with getAllAttributes method.

Reply:
`getFilterAttributes` we already have this method. Can we use it?